<launch>
  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="dev" value="/dev/input/js0" />
    <param name="autorepeat_rate" value="20" />
  </node>

  <node pkg="ezgripper_driver" type="ezgripper.py" name="ezgripper" output="screen">
    <param name="~port" value="/dev/ttyUSB0"/>
    <param name="~baud" value="57600"/>
    <rosparam param="servo_ids">[9]</rosparam>
  </node>
  
  <node pkg="ezgripper_driver" type="client.py" name="ezgripper_client" output="screen"/>

</launch>
